sections/04_study.md

```diff
@@ -56,7 +56,7 @@ approaches applied to gene expression data are powerful methods for
 identifying gene signatures that may otherwise be overlooked.
 An additional benefit of unsupervised approaches is that
 ground truth labels, which are often difficult to acquire or are incorrect, are
-nonessential. However, careful interpretation must be performed regarding how
+nonessential. However, careful interpretation must be performed when
```

Comment:
"the genes that have been aggregated into features must be interpreted carefully"?